handsontable/src/plugins/autoColumnSize/__tests__/autoColumnSize.spec.js

```diff
@@ -1422,7 +1422,23 @@ describe('AutoColumnSize', () => {
     });
   });
 
-  describe('adjust to HyperFormula calculation result', () => {
+  describe('should work together with formulas plugin', () => {
+    it('should calculate widths only once during the initialization of Handsontable with formulas plugin enabled', () => {
+      const columnWidthCalculationSpy = spyOn(
+        Handsontable.plugins.AutoColumnSize.prototype,
+        'addColumnToGhostTableForWidthCalculation'
+      ).and.callThrough();
+
+      handsontable({
+        data: [[42], ['=A1']],
+        formulas: {
+          engine: HyperFormula
+        },
+      });
+
+      expect(columnWidthCalculationSpy).toHaveBeenCalledTimes(1);
+    });
```

Comment:
I found that you can change the approach with mocking the method that we want to make private to one with mocking _GhostTable_ (protected) module.
```suggestion
    it('should calculate widths only once during the initialization of Handsontable with formulas plugin enabled', () => {
      Handsontable.hooks.add('beforeInit', function() {
        spyOn(this.getPlugin('autoColumnSize').ghostTable, 'addColumn').and.callThrough();
      })

      handsontable({
        data: [[42], ['=A1']],
        formulas: {
          engine: HyperFormula
        },
      });

      expect(getPlugin('autoColumnSize').ghostTable.addColumn).toHaveBeenCalledTimes(1);
    });
```

Reply:
Changed